packages/theme/templates/socialMediaIcons/twitch.svg

```diff
@@ -0,0 +1 @@
one" additive="replace" attributeName="points" begin="indefinite" calcMode="spline" dur="250ms" from="16 5 8 13 8 31 14 31 14 36 19 31 23 31 35 19 35 5" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="13 8 8 13 8 31 14 31 14 36 19 31 23 31 32 22 32 8"/><animate fill="freeze" accumulate="none" additive="replace" attributeName="points" begin="indefinite" calcMode="spline" dur="50ms" from="16 5 8 13 8 31 14 31 14 36 19 31 23 31 35 19 35 5" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="13 8 8 13 8 31 14 31 14 36 19 31 23 31 32 22 32 8"/><animate fill="freeze" accumulate="none" additive="replace" attributeName="points" begin="indefinite" calcMode="spline" dur="75ms" from="13 8 8 13 8 31 14 31 14 36 19 31 23 31 32 22 32 8" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="16 5 8 13 8 31 14 31 14 36 19 31 23 31 35 19 35 5"/></path><path fill="#9147FF" d="m637 575 83-84V262H387v313h83v83l84-83z"><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="150ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="250ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="50ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="75ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/></path><path fill="#FFF" d="M512 345h42v125h-42V345zm146 0v125h-42V345h42z"><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="150ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="250ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="50ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="75ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/></path></svg>
```

Comment:
Hi there, you should try to compress these icons, e.g.: remove `xml:space="preserve"` and `<animate>` tag:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" class="icon twitch-icon" viewBox="0 0 1024 1024"><circle cx="512" cy="512" r="512" fill="#9147FF"/><path fill="#FFF" d="M366 220 262 324v376h125v104l104-104h84l187-188V220z"/><path fill="#9147FF" d="m637 575 83-84V262H387v313h83v83l84-83z"/><path fill="#FFF" d="M512 345h42v125h-42V345zm146 0v125h-42V345h42z"/></svg>
```